### DISCUSSION TITLE
How to use black window instead of the window behind when switching?

### DISCUSSION THREAD

**Discussion by WhiteBlackGoose** [i3/i3#5635] [Q&A] [answered]
Hello! Been a user of this amazing WM for quite some time already.

Problem is, sometimes there's quite a lot of load and not all apps manage to redraw instantly. So when for example I switch between workspaces or tabs in i3, first thing that is showing is the previous window (while the one I switched to is redrawing). This is inconvenient because I may have wanted to switch, for example, from a personal messenger app, but while it's redrawing, the app's content is all over the screen. It would be nice to default to black window instead, is it possible?

**Comment by okraits** [comment, marked as answer]
Unfortunately, this is not possible. i3 has no control over the contents of the windows. It only draws the title bar and borders.

**Reply by WhiteBlackGoose** [reply]
I guess, it's a question for X then. Thanks
